typescript/examples/vercel-ai/send-nft-to-twitter/index.ts

```diff
@@ -23,12 +23,16 @@ const walletClient = createWalletClient({
     chain: sepolia,
 });
 
+// TODO(alfonso-paella) Should we document the Crossmint API key requirements?
+// Is there a difference between staging and production keys?
```

Comment:
There is. Mention that you should get a production key for mainnets, and a staging one for testnets